File: C# Pattern Programs/Symbol Patterns/Pattern 74.cs
using System;

class Pattern_208
{
    public static void Main()
    {
        int size = 4;
        for (int i = size; i >= -size; i--)
        {
            for (int j = 0; j <= Math.Abs(i); j++)
            {
                Console.Write("*");
            }
            Console.WriteLine("");
        }
        Console.ReadKey(true);
    }
}

File: JavaPatternPrograms/Symbol Patterns/Pattern 74.java
class Pattern_208
{
    public static void main(String args[])
    {
        int size = 4;
        for (int i = size; i >= -size; i--)
        {
            for (int j = 0; j <= Math.abs(i); j++)
            {
                System.out.print("*");
            }
            System.out.println("");
        }
    }
}